pkg/jobrunaggregator/jobrunaggregatorlib/ci_data_client.go

```diff
@@ -112,11 +140,13 @@ LIMIT 1
 func (c *ciDataClient) GetLastJobRunWithDisruptionDataForJobName(ctx context.Context, jobName string) (*jobrunaggregatorapi.JobRunRow, error) {
 	// the JobRun.Name is always increasing, so we can sort by that name.  The starttime is based on the prowjob
 	// time and I don't think that is coordinated.
+	// the disruption jobrun table is now distinct, so we will use the disruption jobrun table as authoritative for what data should and should not
+	// be uploaded rather than using the absence of backenddisruption itself.  Some jobs don't include this data so we end up
+	// inserting many duplicated entries
```

Comment:
Can you clarify the last sentence. "Some jobs do not include disruption data, those that do will be duplicated in both the backend disruption job run table, as well as the main job run table."?